src/collection/collection.service.ts

```diff
@@ -324,6 +326,19 @@ export class CollectionService {
 
 		return omit(collection, propertiesToDelete);
 	}
+	public static async getBundles(limit: number): Promise<Avo.Collection.Collection[]> {
```

Comment:
kan je nog een await toevoegen in src/collection/components/modals/AddToBundleModal.tsx regel 248
ik ben die vergeten en anders wordt die exceptie niet opgevangen